src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -306,6 +307,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ),
                     'required' => false,
                     'invalid_message' => $this->trans('This field is invalid.', 'Admin.Notifications.Error'),
+                    'constraints' => [
+                        new GreaterThan([
+                            'value' => 0,
+                            'message' => $this->trans(
+                                'Invalid value: the maximum number of payment days must be a positive amount.',
```

Comment:
```suggestion
                                '%s is invalid. Please enter an integer greater than or equal to 0.',
```

We could use this existing string. :-)

Reply:
Fixed